pkg/api/graph/graphview/veneering_test.go

```diff
@@ -259,38 +259,35 @@ func TestGraph(t *testing.T) {
 	})
 	deploygraph.EnsureDeploymentConfigNode(g, &deployapi.DeploymentConfig{
 		ObjectMeta: kapi.ObjectMeta{Namespace: "other", Name: "deploy1"},
-		Triggers: []deployapi.DeploymentTriggerPolicy{
-			{
-				ImageChangeParams: &deployapi.DeploymentTriggerImageChangeParams{
-					From:           kapi.ObjectReference{Namespace: "default", Name: "other"},
-					ContainerNames: []string{"1", "2"},
-					Tag:            "tag1",
+		Spec: deployapi.DeploymentConfigSpec{
+			Triggers: []deployapi.DeploymentTriggerPolicy{
+				{
+					ImageChangeParams: &deployapi.DeploymentTriggerImageChangeParams{
+						From:           kapi.ObjectReference{Kind: "ImageStreamTag", Namespace: "default", Name: "other:tag1"},
```

Comment:
I guess alternatively, was this simply an invalid construct that could never have been created in the external version of the API?